### PR TITLE
Ignore adding additional optIns also for "jvmCoreMain" source set

### PR DESCRIPTION
**Pull request by Tapchicoma**
After changes required for https://youtrack.jetbrains.com/issue/KT-57292/Rework-configuration-of-compiler-settings-in-MPP-Projects consistency checks were move later in configuration phase. Now they started also catch inconsistency between "jvmMain" and "jvmCoreMain" source sets.

### CODE REVIEW

**Comment by Tapchicoma** [comment]
Fixes build failure in [this MR](https://jetbrains.team/p/kt/reviews/11269/timeline)

**Review by qwwdfsad — APPROVED** [review]

**Comment by qwwdfsad** [comment]
Could you please target `develop` branch though?
That way there won't be heisen-commits across mainline and aggregator branches. Release manager will take care of it after the merge

**Comment by Tapchicoma** [comment]
Oh, I will rebase it now.

**Comment by mshishkina** [comment]
Merged in kotlin-community/dev branch.

**Pull request merged**
